src/core/libraries/kernel/thread_management.cpp

```diff
@@ -318,7 +318,10 @@ int PS4_SYSV_ABI scePthreadAttrGetstackaddr(const ScePthreadAttr* attr, void** s
         return SCE_KERNEL_ERROR_EINVAL;
     }
 
-    int result = pthread_attr_getstackaddr(&(*attr)->pth_attr, stack_addr);
+    size_t stack_size = 0;
+    pthread_attr_getstacksize(&(*attr)->pth_attr, &stack_size);
```

Comment:
This getstacksize call is superfluous I think

Reply:
You're right, getstack would set the stack size just like getstacksize would